inc/com/centreon/engine/nebcallbacks.hh

```diff
@@ -68,7 +68,9 @@
 #define NEBCALLBACK_ADAPTIVE_ESCALATION_DATA 40
 #define NEBCALLBACK_ADAPTIVE_TIMEPERIOD_DATA 41
 
-#define NEBCALLBACK_NUMITEMS 42 /* Total number of callback types we have. */
+#define NEBCALLBACK_ENGINERPC 42
```

Comment:
is this needed ?

Reply:
I would like to say no, but we have a generic code that is not really ... generic... :smiley: